_delphi_utils_python/delphi_utils/validator/dynamic.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, params):
                                                common_params["span_length"]),
             generation_date=date.today(),
             max_check_lookbehind=timedelta(
-                days=dynamic_params.get("ref_window_size", 7)),
+                days=dynamic_params.get("ref_window_size", 14)),
```

Comment:
Instead of doing the `max` procedure on both lines 246 and 599, maybe we could do it here. That means we could get rid of the `semirecent_lookbehind` variable entirely, which would simplify things a bit more.